demos/inpage/demo/LinksPage.tsx

```diff
@@ -16,26 +16,38 @@ const addFundsDefault = (() => {
 
 const LinksPage = () => {
   const demo = DemoContext.use();
-  const address = demo.useAddress();
   const [, setPath] = usePath();
+  const account = demo.useAccount();
 
   return (
     <div className="links-page">
       <Button
         secondary
-        disabled={address === undefined}
+        disabled={account === undefined}
         onPress={async () => {
-          if (address === undefined) {
+          if (account === undefined) {
             return;
           }
 
           const admin = await demo.waxInPage.requestAdminAccount(
             'fund-new-account',
           );
 
+          const { ownerAddress } = account.toData();
+
+          await (
+            await admin.sendTransaction({
+              to: account.address,
+              value: ethers.parseEther(
+                config.addFundsEthAmount ?? addFundsDefault,
+              ),
+            })
+          ).wait();
+
+          // TODO: (merge-ok) ensure wallet is already funded
```

Comment:
we also need to fund the owner of the wallet as the owner needs to call "enable module" and pay for that transaction